internal/adapters/api/handler/role_permissions.go

```diff
@@ -10,46 +10,69 @@ import (
 
 type rolePermissionsHandler struct {
 	rolePermissionsService ports.RolePermissionsService
-	log                    ports.Logger
+	logger                 ports.Logger
 }
 
-func NewRolePermissionsHandler(rolePermissionsService ports.RolePermissionsService, log ports.Logger) ports.RolePermissionsHandler {
+func NewRolePermissionsHandler(rolePermissionsService ports.RolePermissionsService, logger ports.Logger) ports.RolePermissionsHandler {
 	return &rolePermissionsHandler{
 		rolePermissionsService: rolePermissionsService,
-		log:                    log,
+		logger:                 logger,
 	}
 }
 
 func (h *rolePermissionsHandler) AddPermissionsToRole(c *gin.Context) {
 	rolePermissions := domain.RolePermissions{}
```

Comment:
declare variable without initialising it

Reply:
Done